sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ObjectExpressionsSuite.scala

```diff
@@ -424,7 +424,7 @@ class ObjectExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
   test("LambdaVariable should support interpreted execution") {
     def genSchema(dt: DataType): Seq[StructType] = {
       Seq(StructType(StructField("col_1", dt, nullable = false) :: Nil),
-        StructType(StructField("col_1", dt, nullable = true) :: Nil))
```

Comment:
I think you can revert these, but I don't feel strongly about it

Reply:
I see. I also reverted other two redundant default parameters.